class-vip-support-user.php

```diff
@@ -400,6 +413,62 @@ public function filter_wp_redirect( $location ) {
 		return $location;
 	}
 
+	/**
+	 * Filters a users email address, removing the username if the email
+	 * is a VIP Support email alias.
+	 *
+	 * This helps simplify support email aliases in the users list so
+	 * they're not as lengthy and easier to understand at a glance.
+	 *
+	 * @param $email string The email address of a user.
+	 *
+	 * @return string
+	 */
+	public function filter_vip_support_email_aliases( $email ) {
+		if ( is_admin() && $this->is_vip_support_email_alias( $email ) ) {
+			return self::VIP_SUPPORT_EMAIL_ADDRESS;
+		}
+		return $email;
+	}
+
+	/**
+	 * Filters a users Gravatar URL, altering it if the email is a VIP
+	 * Support email alias.
+	 *
+	 * Since the email aliases are fake, they don't have a Gravatar.
+	 * Instead of showing the mystery man, get the Gravatar for the
+	 * real `VIP_SUPPORT_EMAIL_ADDRESS` email address.
+	 *
+	 * @param $url         string The Gravatar url.
+	 * @param $id_or_email mixed  The users ID, email, or a WP_User, WP_Post, WP_Comment object.
+	 * @param $args        array  Arguments passed to get_avatar_url(), after processing.
+	 *
+	 * @return string
+	 */
+	public function filter_vip_support_email_gravatars( $url, $id_or_email, $args ) {
+
+		if ( ! is_admin() ) {
+			return $url;
+		}
+
+		// Get the user's email address.
+		if ( is_numeric( $id_or_email ) ) {
+			$user       = get_user_by( 'id', $id_or_email );
+			$user_email = $user->user_email;
+		} elseif ( is_string( $id_or_email ) ) {
+			$user_email = $id_or_email;
+		} elseif ( $id_or_email instanceof WP_User ) {
+			$user_email = $id_or_email->user_email;
+		}
+
+		if ( isset( $user_email ) && $this->is_vip_support_email_alias( $user_email ) ) {
+			$email_hash = md5( self::VIP_SUPPORT_EMAIL_ADDRESS );
```

Comment:
This isn't going to change, so we can just hard-code it in a class `const` as well: `https://secure.gravatar.com/avatar/c83fd21f1122c4d1d8677d6a7a1291d3`

And we can just append the size below.

Reply:
Great point! Adjusted in https://github.com/Automattic/vip-support/pull/73/commits/ca7fa65ad9c5fb1db6d2233c965c53864d4d6e67

Reply:
ca7fa65 has added the hard-coded URL, but this `$email_hash` is still being assigned to even though it's unused. This line 470 can be deleted.